### DISCUSSION TITLE
create customer from administrator (and vice versa)

### DISCUSSION THREAD

**Discussion by alckp** [vendurehq/vendure#1052] [Q&A]
Hi, 

When we create an administrator, a user entity is created and associated to the administator . 
But if I try to create an customer with the same email, an error occured "email already used".
I need an administrator to also be a customer.
How should I do to create an administrator from an existing user entity or may be, could I by-pass this email checking to create 2 different users ?

**Comment by michaelbromley** [comment]
Hi,

Currently this is not supported. You could open a feature request for this and it will be considered for a future release.
